manifests/forward.pp

```diff
@@ -5,6 +5,7 @@
 define unbound::forward (
   $address,
   $zone = $name,
+  $forward_first = 'no'
```

Comment:
Can do. Is input validate being done elsewhere through the stdlib functions? Will I need to update the requirements section under README.md?